pyaedt/modeler/stackup_3d.py

```diff
@@ -950,6 +996,16 @@ def plating_ratio(self, val):
         else:
             self._app.logger.error("Plating has to be between 0 and 1")
```

Comment:
```suggestion
            self._app.logger.error("Plating has to be between 0 and 1.")
```

Reply:
Messages are to be complete sentences, including punctuation. I'm not sure if this change will impact a test though.